code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm

```diff
@@ -293,8 +293,8 @@
 
 /datum/reagent/medicine/c2/seiver //a bit of a gray joke
 	name = "Seiver"
-	description = "A medicine that shifts functionality based on temperature. Hotter temperatures will remove amounts of toxins, while coder temperatures will heal larger amounts of toxins only while the patient is irradiated. Damages the heart." //CHEM HOLDER TEMPS, NOT AIR TEMPS
-	var/radbonustemp = (T0C - 100) //being below this number gives you 10% off rads.
+	description = "A medicine that shifts functionality based on temperature. Hotter temperatures will heal more toxicity, while colder temperatures will heal larger amounts of toxicity but only while the patient is irradiated. Damages the heart." //CHEM HOLDER TEMPS, NOT AIR TEMPS
+	var/radbonustemp = 100 //being below this number gives you radiation healing.
```

Comment:
While you're in the area and on your conquest against shitcode, please move the definition of `var/radbonustemp` to be at the bottom of the typepath declaration, after all of the vars we inherit from parent. New vars go after old vars for cleanliness.

Also, transform this comment into an autodoc, like:
```txt
/// Add pertinent documentation as to what this variable means/does.
var/radbonustemp
```

Just to be kind to future explorers of this code. You can learn more about DMDoccing here: https://github.com/tgstation/tgstation/blob/master/.github/guides/AUTODOC.md

(also, `radbonustemp' as one word makes my head hurt. could you kindly change it to a better name that fits it's use, and have it be snake case (i.e. `rad_bonus_temp`, but that doesn't sound right? Just make it more descriptive overall)).